src/test/java/com/redhat/devtools/lsp4ij/features/formatting/TypeScriptClientSideFormatOnStatementTerminatorTest.java

```diff
@@ -763,8 +812,8 @@ export class Foo {
                 "[]",
                 mockRangeFormattingJson,
                 clientConfiguration -> {
+                    clientConfiguration.statementTerminatorCharacters = ";";
```

Comment:
And because the statement terminator is no longer hard-coded as `;`, we have to configure it explicitly for these tests.